server/liveblog/themes/themes_assets/default/templates/template-poll-utils.html

```diff
@@ -0,0 +1,30 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/moment.min.js"></script>
```

Comment:
We already have `moment.js` as dependency in the theme. Why do you need it again like this here?